integration-tests/helpers.js

```diff
@@ -85,6 +86,17 @@ class FakeAgent extends EventEmitter {
 
 function spawnProc (filename, options = {}) {
   const proc = fork(filename, options)
+  if (proc.stdout) {
+    const readline = rl.createInterface(proc.stdout)
+    readline.on('line', line => {
+      try {
+        const { traces } = JSON.parse(line)
+        if (traces) proc.emit('logMessage', { log: traces })
```

Comment:
Can we somehow make this more robust? What if we test for example that a custom JSON logger is used by the tracer when configured? How can we validate that the JSON is actually traces? I'm also not sure that this is the right place for this parsing in any case.

Reply:
Right, I'm thinking the correct thing to do is just send log lines as events, and have any parsing be done in test code. That should be better for clarity and make it more adaptable for plugin tests or alternative logger tests later on.